### ISSUE TITLE
Is caption dropout possible for WAN training?

### ISSUE THREAD

**Issue by choowkee** [kohya-ss/musubi-tuner#434]
I've seen caption dropout as a option for training WAN loras using AI-toolkit. And was wondering if this is something that can also be done in musubi? I believe sd-scripts supports caption_dropout_rate for images already

**Comment by kohya-ss** [comment]
Yes, it is technically possible.

However, I'm a little skeptical about how effective caption dropout is. Has its effectiveness been confirmed? If so, I'll consider adding it.

**Comment by Pashahlis** [comment]
> Yes, it is technically possible.
> 
> However, I'm a little skeptical about how effective caption dropout is. Has its effectiveness been confirmed? If so, I'll consider adding it.

I used to use it all the way back in the 1.5 and XL days, but from FLUX onwards I found it better to use none at all. However, if you have nothing else to do (lol, doubt) you could add it so that I may be able to test its effectiveness. For both WAN and Qwen.

**Comment by choowkee** [comment]
> Yes, it is technically possible.
> 
> However, I'm a little skeptical about how effective caption dropout is. Has its effectiveness been confirmed? If so, I'll consider adding it.

Thank you for the reply. I think it could help with generalization for character video Loras in WAN. 

I switched over to Musubi from AI-Toolkit and I trained the same exact character Lora with all the same training parameters/datasets. Caption dropout is one of the few options I could not replicate in Musubi. For reference I used 0,05 caption_dropout_rate (5%) in AI-Toolkit training.

After testing T2V generations with the Loras trained with Musubi it appears that the Loras learn from training captions very strictly (which of course makes sense since all image-training pairs are taking into account during training) which could get in the way of generalization when it comes maintaining character likeness, especially when using natural language captioning.

But I am not an expert on this topic, its just my personal observation & opinion - I apologize if this is not the best example. I will try to train my next Lora by adding images/videos without captions - I believe this should help simulate caption_dropout functionality in Musubi.

(Btw I still have very good results in Musubi, thank for your support!)

**Comment by Pashahlis** [comment]
@kohya-ss @choowkee I was training just fine using no caption dropout in WAN2.1. Little to no overfitting happened.

But now in WAN2.2 and Qwen I have the same issues with overfitting. I have tried everything in the book so far to fix it using just the available training settings but to no avail. "Lower intensity training" just does not seem to fix it, while likeness obviously takes a hit.

I dont know if caption dropout will fix this at all, because I did not have that experience in FLUX either, but I just wanna test to see if it might. I guess I could also just test AI-Toolkit for a second, but that also lacks other options again that Musubi has lol and I prefer Musubi.

**Comment by zhaol4v** [comment]
I used 5% caption dropout kohya's sdxl and flux scripts to successfully overfitting, so it would be nice to try it for wan2.2
